mlir/lib/CAPI/Dialect/MIGraphX.cpp

```diff
@@ -33,7 +33,8 @@ void mlirGetKernelInfo(MlirModule module, int *size, void *data) {
   int argIdx = 0;
   llvm::StringRef kernelName;
 
-  assert (size != nullptr || data != nullptr); // either of ptrs should be provided.
+  assert(size != nullptr ||
```

Comment:
Can we go ahead and make this a noisy assert? `assert((size != nullptr || data != nullptr) && "Either size or data pointer should be provided)`

Reply:
I'm merely fixing `clang-format` here. To improve the quality of the assert I think it's up to the owner of the module, @jungpark-mlir ?

Reply:
Yeah, that change (embedding comment) looks better